recipe/meta.yaml

```diff
@@ -157,6 +155,11 @@ outputs:
         - libcurand-dev                         # [build_platform != target_platform]
         - libcusolver-dev                       # [build_platform != target_platform]
         - libcusparse-dev                       # [build_platform != target_platform]
+        # optional dependencies for CUDA 12+
+        - cusparselt ~=0.6.0                    # [build_platform != target_platform]
+        {% else %}
+        # optional dependencies for CUDA 11
+        - cusparselt ~=0.2.0                    # [build_platform != target_platform]
```

Comment:
cuSPARSELt dropped CUDA 11 support in [the 0.5.0 release]( https://docs.nvidia.com/cuda/cusparselt/release_notes.html#cusparselt-v0-5-0 )

There were various issues getting the previous versions working. So just used 0.2.0 for cuSPARSELt on CUDA 11 (as was done before)